lib/application/pages/details/collection_details_page.dart

```diff
@@ -4,10 +4,10 @@ import 'package:flutter_riverpod/flutter_riverpod.dart';
 import 'package:layoutr/common_layout.dart';
```

Comment:
Sorted the imports alphabetically (`flutter analyze` complains if not)

Reply:
Do you know if this is linteable? It seems that it should, if flutter is complaining.

Reply:
This is being catch by the linter:

![CleanShot 2021-07-28 at 17 26 14](https://user-images.githubusercontent.com/11745745/127391016-39f8a135-6c9d-4646-99fc-1bed44293cf9.png)

I'm just thinking why Flutter wasn't complaining before about it, but I have no idea why